kore/src/Kore/ASTVerifier/DefinitionVerifier.hs

```diff
@@ -100,58 +99,57 @@ verifyAndIndexDefinition builtinVerifiers definition = do
             definition
     return indexedModules
 
-{-|Verifies a `ParsedDefinition` against a preverified definition, consisting of
+{- |Verifies a `ParsedDefinition` against a preverified definition, consisting of
 map of indexed modules and a map of defined names.
 
 If verification is successfull, it returns the updated maps op indexed modules
 and defined names.
 -}
-verifyAndIndexDefinitionWithBase
-    ::  (Map ModuleName VerifiedModule', Map Text AstLocation)
-    ->  Builtin.Verifiers
-    ->  ParsedDefinition
-    ->  Either (Error VerifyError)
-            (Map ModuleName VerifiedModule', Map Text AstLocation)
+verifyAndIndexDefinitionWithBase ::
+    (Map ModuleName VerifiedModule', Map Text AstLocation) ->
+    Builtin.Verifiers ->
+    ParsedDefinition ->
+    Either
+        (Error VerifyError)
+        (Map ModuleName VerifiedModule', Map Text AstLocation)
 verifyAndIndexDefinitionWithBase
     alreadyVerified
     builtinVerifiers
-    definition
-  = do
-    let
-        (verifiedModulesCache, baseNames) =
-            (implicitModules, implicitNames) <> alreadyVerified
+    definition =
+        do
+            let (verifiedModulesCache, baseNames) =
+                    (implicitModules, implicitNames) <> alreadyVerified
```

Comment:
Here is another example of creeping indentation, but this function should also probably be rewritten. (So it does not use an explicit fold.)